server/conf/application.conf

```diff
@@ -598,3 +598,10 @@ api_applications_list_max_page_size = ${?CIVIFORM_API_APPLICATIONS_LIST_MAX_PAGE
 # In development features.
 application_status_tracking_enabled = false
 application_status_tracking_enabled = ${?CIVIFORM_APPLICATION_STATUS_TRACKING_ENABLED}
+
+# If enabled allows for setting of feature flags via HTTP.
```

Comment:
Agree with this sentiment entirely.

To further reduce the chances that this is accidentally set, I'd set the entry to false here and allow the environment variable override from application.dev.conf and application.test.conf.

Something I think that would make the most sense long-term is for us to have an application.prod.conf and always set this to false there.

Reply:
Great idea, moved it.